infra/stack.yaml

```diff
@@ -50,8 +33,8 @@ Resources:
     Properties:
       TemplateURL: !Sub 'https://s3.amazonaws.com/seattle-uat-cftmpl/${Timestamp}/database.yaml'
       Parameters:
-        DBUsername: !Ref 'DBUsername'
-        DBPassword: !Ref 'DBPassword'
+        DBUsername: uat
```

Comment:
**Suggestion** (if-minor): it would be nice if we can still have a variable for the database username and reference it in here and below so we don't need to change two places if we ever want to change them.

Reply:
Done!

Reply:
Would you mind referencing back to the variable? Thanks.